### PR TITLE
4.9 config settings

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2106,7 +2122,7 @@ Enable Server-Side Encryption for Amazon S3
 
 Enable Amazon S3 Debugging
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-**True**: Additional debugging information is included in the system logs. Typically set to `false` in production.
+**True**: When true, log additional debugging information to the system logs. Typically set to `false` in production.
 
 **False**: No Amazon S3 debugging information is included in the system logs.
```
Comment (amyblais) [inline source/administration/config-settings.rst]: Not sure how to add "Test Connection" button.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2130,6 +2139,9 @@ Enable Amazon S3 Debugging
 | This feature's ``config.json`` setting is ``"AmazonS3Trace": false`` with options ``true`` and ``false`` for above settings respectively.                            |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Test Connection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (amyblais) [inline source/administration/config-settings.rst]: There was no help text in the Test Connection button, what do I write here as a description?

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -1655,6 +1655,22 @@ The password associated with the SMTP username.
 | This feature's ``config.json`` setting is ``"SMTPPassword": ""`` with string input.                                                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Customer Type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+XXXX///
```
Comment (amyblais) [inline source/administration/config-settings.rst]: Will need help with a description for this and the one below.

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -1323,6 +1321,22 @@ Login Button Text
 | This feature's ``config.json`` setting is ``"LoginButtonText": ""`` with string input.                                                                               |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Scoping IDP Provider Id
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Allows an authenticated user to skip the initial login page of their federated Azure AD server, and only require a password to log in. (XXXX/// optional?)
```
Comment (esethna) [inline source/administration/config-settings.rst]: can we  remove note?

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -1625,6 +1639,14 @@ Enable SMTP Authentication
 | This feature's ``config.json`` setting is ``"EnableSMTPAuth": false`` with options ``true`` and ``false`` for above settings respectively.                           |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Customer Type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Type of Global Relay customer account your organization has.
```
Comment (esethna) [inline source/administration/config-settings.rst]: We should list the options and describe them. Similar to other settings

File: source/administration/config-settings.rst [hunk 6]
```diff
@@ -1785,6 +1826,8 @@ Enable OAuth 2.0 Service Provider
 
 Restrict managing integrations to Admins
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*Removed in April 16th, 2018 release*
```
Comment (esethna) [inline source/administration/config-settings.rst]: I don't think this is being removed, can we confirm?

File: source/administration/config-settings.rst (hunk 2 above)
Comment (esethna) [reply source/administration/config-settings.rst]: The button just makes sure we can access the server and the settings are valid

File: source/administration/config-settings.rst [hunk 7]
```diff
@@ -3630,3 +3676,15 @@ Select the themes that can be chosen by users when ``"EnableThemeSelection"`` is
...
 | This feature's ``config.json`` setting is ``"AllowedThemes": "default"`` with options ``default``, ``organization``, ``mattermostDark`` and ``windows10`` optionally separated by commas. For example, ``["mattermostDark", "windows10"]`` |
 +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Display Settings  (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Timezone
+Select the timezone used for timestamps in the user interface and email notifications.
+
+XXXXX///
```
Comment (esethna) [inline source/administration/config-settings.rst]: When it's false it hides the timezone setting in Account settings. When true it's shows the setting (so users can set their preferred timezone), but also automatically assigns a timezone the next time users connect.

**Pull request merged**